versions/ufo3/glyphs/glif.md

```diff
@@ -167,6 +167,18 @@ This element may occur any number of times.
 
 #### This element has no child elements.
 
+#### Anchor naming conventions
+
+The following are agreed upon standard naming conventions for the `name` of anchors for special uses.
+
+##### hcaret and vcaret
+
+Anchors with a `name` starting with `hcaret` or `vcaret` indicate horizontal or vertical position for the [GDEF Ligature Caret List table]. Anchors with the name `hcaret` provide the `x` value for a ligature caret, and the `y` value is ignored. Likewise, anchors with the name `ycaret` provide the `y` values for a ligature caret, and the `x` value is ignored.
+
+Both names may be followed by an underscore and arbitrary text (`_X`) for designers to have unique caret position names. The order of carets written to the GDEF table is based upon ascending value order, the `_X` text is not taken into account.
+
+`hcaret` and `vcaret` must not both occur in a glyph.
+
```

Comment:
Maybe mention that it is expected to have N-1 caret positions for N ligature components.